### PR TITLE
[release] Include JDBC drivers built from source

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution-no-drivers.xml [hunk 1]
```diff
@@ -19,8 +19,6 @@
               <exclude>com.fasterxml.jackson.core:jackson-core:*</exclude>
 
               <!-- Exclude connection drivers -->
-              <exclude>mysql:mysql-connector-java:*</exclude>
```
Comment (gunnarmorling) [inline debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution-no-drivers.xml]: I don't quite get this, won't it re-addd the upstream drivers?

File: pom.xml [hunk 2]
```diff
@@ -68,7 +68,7 @@
         <!-- Databases -->
         <version.postgresql.driver>42.2.5</version.postgresql.driver>
         <version.mysql.server>5.7</version.mysql.server>
-        <version.mysql.driver>8.0.16</version.mysql.driver>
+        <version.mysql.driver>8.0.17</version.mysql.driver>
```
Comment (gunnarmorling) [inline pom.xml]: That'd warrant a JIRA issue for the purposes of changelog mention.

Comment (jpechane) [reply pom.xml]: @gunnarmorling Thanks for spotting, this is test leftover

File: debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution-no-drivers.xml (hunk 1 above)
Comment (jpechane) [reply debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution-no-drivers.xml]: @gunnarmorling No, upstream build will use upstream drivers, prodcut build will use product drivers - the GAV is automatically modified by build system.

Comment (gunnarmorling) [reply debezium-assembly-descriptors/src/main/resources/assemblies/connector-distribution-no-drivers.xml]: Ok, got it.

**Comment by gunnarmorling** [comment]
Thanks, @jpechane!

**Pull request merged**